Snakefile-sorting-hat

```diff
@@ -142,6 +150,16 @@ rule get_experiment_metadata:
             >&2 echo "Can't retrieve organism: neither {wildcards.accession}/{wildcards.accession}.condensed-sdrf.tsv nor {wildcards.accession}/{wildcards.accession}-configuration.xml not found "
             exit 1
         fi
+        # fix species name
+        #organism_check=$( python3  {workflow.basedir}/bin/bulk_recalculations_check_species_names.py $(echo $organism) {params.check_species_file} )
+        while read line; do
+        echo "$line"
+        DIR=$line
+        done < <(perl {workflow.basedir}/bin/bulk_recalculations_check_species_names.pl $(echo $organism) {params.check_species_file}  )
+	    organism_check=$(echo $DIR)
+
+        echo $organism "check-> "$organism_check >> organisms.txt
```

Comment:
Do we need this organisms.txt file there?

Reply:
Not really, only useful for testing. I'll comment it for now